css-fonts/styles.css

```diff
@@ -0,0 +1,10 @@
+body {
+  font-family: 'Rubik', sans-serif;
+  font-family: 'Teko', sans-serif;
+  line-height: 25px;
+}
+
+h1 {
+  font-family: 'Rubik', sans-serif;
+  font-family: 'Teko', sans-serif;
```

Comment:
By doing this twice, you are just overwriting the line above. You can only declare a property in any CSS rule once.